### PR TITLE
Update Set-SPOTenant.md

### PR DESCRIPTION
**Pull request by timwoo187**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -2018,6 +2020,42 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
 
+### -ReduceTempTokenLifetimeEnabled 
+Enables reduced session timeout for temporary URLs used by apps for document download scenarios. Reduction occurs when app’s redeeming IP address does not match the original requesting IP. Default value is 15 minutes if ReduceTempTokenLifetimeValue is not set.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2024]:
```suggestion
Enables reduced session timeout for temporary URLs used by apps for document download scenarios. Reduction occurs when an app redeeming an IP address does not match the original requesting IP. The default value is 15 minutes if ReduceTempTokenLifetimeValue is not set.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 2]
```diff
@@ -2018,6 +2020,42 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
...
+
+> [!NOTE]
+> Reducing this value may bring degradation in end user experience by requiring frequent authentication prompts to users.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2027]:
```suggestion
**Note**: Reducing this value may bring degradation in end-user experience by requiring frequent authentication prompts to users. 
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md (hunk 1 above)
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2024]:
```suggestion

Enables reduced session timeout for temporary URLs used by apps for document download scenarios. Reduction occurs when an app redeeming an IP address does not match the original requesting IP. The default value is 15 minutes if ReduceTempTokenLifetimeValue is not set.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md (hunk 2 above)
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2027]:
```suggestion
**Note**: Reducing this value may bring degradation in end-user experience by requiring frequent authentication prompts to users. 
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 3]
```diff
@@ -2018,6 +2020,42 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
...
+Aliases:
+Applicable: SharePoint Online
+Required: False
+Position: Named
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+### -ReduceTempTokenLifetimeValue
+Optional parameter to set the session timeout value for temporary URLs. The value can be between 5-15 minutes and the default value is 15 minutes.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2043]:
```suggestion

Optional parameter to set the session timeout value for temporary URLs. The value can be set between 5 and 15 minutes and the default value is 15 minutes.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @timwoo187, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by tseward** [comment]
@timwoo187 could we have some verbiage added here or a link to another article that describes "temporary URLs"?

**Review by get-itips — COMMENTED** [review]
agree with @tseward

**Comment by yogkumgit** [comment]
@timwoo187 Please update. Thanks

**Comment by yogkumgit** [comment]
@timwoo187 Gentle reminder.

**Comment by timwoo187** [comment]
Temporary URLs are used with certain client scenarios similar to WAC token lifetime.

**Comment by yogkumgit** [comment]
@dariomws @tseward

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws

**Comment by yogkumgit** [comment]
@dariomws Gentle reminder.

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**